src/SearchField/SearchField.scss

```diff
@@ -1,14 +1,31 @@
+@import "variables";
+
 .pgn__searchfield {
-    @extend .border;
-    @extend .rounded;
     transition: $input-transition;
+    border: $search-border-width solid $search-border-color;
+
+    &:disabled {
+        opacity: $search-disabled-opacity;
```

Comment:
Does this need to be `.disabled` so it applies as a className vs. a pseudoselector? I'm also curious if this is the most appropriate way to disable these elements from an a11y perspective, as the `input` and `button` elements do take a `disabled` prop natively with their own styles like a faded opacity built in.

Does it make sense to also pass `disabled` prop to these input/button elements as well?

Reply:
Yes, I see the disable functionality was made with the `.disabled` className and then I removed it by mistake after one of the code review.
- For the whole component `disabled` works good now.
- For input I added prop `option: { disabled: true/false }` but I investigated and it doesn't implemented yet. But I pass this prop anyway, so if Input have disabled functionality in future it will use them immediately.
- For the SubmitButton I added disabled prop from the context. Now button is properly disabled.